API/Modules.md

```diff
@@ -14,12 +14,12 @@ The best place to find API documentation for a Tessel module is through the link
 
 ## Quick links
 
-* [See all the modules and find their API documentation](//tessel.io/modules)
-* [Learn how to use modules on the start page](//start.tessel.io/modules)
+* [See all the modules and find their API documentation](https://tessel.io/modules)
```

Comment:
Why force https here? The double slash should keep whatever you currently have (if http, stay http), then I think we force https from our own http site.